tools/cli/go-whisk-cli/main.go

```diff
@@ -48,6 +48,7 @@ func init() {
 
     // Rest of CLI uses the Properties struct, so set the build time there
     commands.Properties.CLIVersion = CLI_BUILD_TIME
+    os.Setenv("CLI_Version", CLI_BUILD_TIME)
```

Comment:
I would either need to inject the CLIVersion properly into ```cli/go-whisk-cli/commands/property.go``` then import this into ```/cli/go-whisk/whisk/client.go```, or just import ```cli/go-whisk-cli/main.go``` into ```/cli/go-whisk/whisk/client.go```.

Not the greatest options imo